playbooks/dependencies-fedora-restricted.yaml

```diff
@@ -14,6 +14,11 @@
 # limitations under the License.
 #
 
+- name: Install python3-dnf
```

Comment:
Perhaps add a note why this was added for rawhide, so that the task can be removed/adjusted at the next fedora release.

Reply:
Good point.  I already added a detailed note to the commit message of the [commit](https://github.com/containers/toolbox/commit/96791726a35d9a19c53064bbb17204f4aba92e46) from @danpawlik 

I got a bit impatient while his pull request https://github.com/containers/toolbox/pull/1338 was going through CI, and rebased this PR on top of his.  That's why it looks a bit confusing.